torchdynamo/utils.py

```diff
@@ -416,6 +416,64 @@ def check_constant_args(args, kwargs):
     return all(x.is_python_constant() for x in itertools.chain(args, kwargs.values()))
 
 
+def check_unspec_python_args(args, kwargs):
+    from .variables.constant import ConstantVariable
+    from .variables.tensor import FakeItemVariable
+    from .variables.tensor import UnspecializedPythonVariable
+
+    return (
```

Comment:
Is there a way to write this without iterating over args + kwargs 3 times?

Reply:
There is, but I don't think this is a bottleneck. There are many other places we are using this way, so maybe we can do a sweep together if we find this is bottleneck.

Reply:
Let's write things efficiently if possible the first time around - its not a bottleneck, but if we can turn 3 iterations over the same thing into one, why not?